website/templates/public/pages/faq.mako

```diff
@@ -118,7 +118,7 @@
 <p>If the project is private, claim the account, and then follow the steps above.</p>
 
 <h3>How do I make folders to organize the files in my project?</h3>
-<p>Components act like folders. Thus, to organize your files into folders, create components. From the project overview page, click “create component” on the right half of the page.</p>
+<p>You can create a folder or a component to help organize your files. To create a folder, navigate to the files tab of your project. Find the storage add-on you'd like to put a folder in and click the "+" sign next to its name. Components also act like folders, but can be used to control privacy settings independently of the whole project. To create a component, navigate to the project overview page and click “create component” on the right half of the page.</p>
```

Comment:
looks good

On Fri, May 8, 2015 at 12:53 PM, saradbowman <notifications@github.com>
wrote:

> In website/templates/public/pages/faq.mako
> <https://github.com/CenterForOpenScience/osf.io/pull/2662#discussion_r29956001>
> :
>
> > @@ -118,7 +118,7 @@
> >  <p>If the project is private, claim the account, and then follow the steps above.</p>
> >
> >  <h3>How do I make folders to organize the files in my project?</h3>
> > -<p>Components act like folders. Thus, to organize your files into folders, create components. From the project overview page, click “create component” on the right half of the page.</p>
> > +<p>You can create a folder or a component to help organize your files. To create a folder, navigate to the files tab of your project. Find the storage add-on you'd like to put a folder in and click the "+" sign next to its name. Components also act like folders, but can be used to control privacy settings independently of the whole project. To create a component, navigate to the project overview page and click “create component” on the right half of the page.</p>
>
> Proposed language changes: FAQs: What is the difference between a
> component and a folder? A folder can be used to organize files within a
> project or component - just like a folder on your own computer groups files
> together. A component also groups files together, but has added
> functionality. Components have privacy and sharing settings independent of
> the overarching project's settings, their own unique, persistent
> identifiers for citation, and their own wiki and add-ons.
> … <#14d34729d88c19a7_>
> On Fri, May 8, 2015 at 10:45 AM, Brian J. Geiger <notifications@github.com>
> wrote: In website/templates/public/pages/faq.mako <
> https://github.com/CenterForOpenScience/osf.io/pull/2662#discussion_r29945131>
> : > @@ -118,7 +118,7 @@ > <p>If the project is private, claim the account,
> and then follow the steps above.</p> > > <h3>How do I make folders to
> organize the files in my project?</h3> > -<p>Components act like folders.
> Thus, to organize your files into folders, create components. From the
> project overview page, click “create component” on the right half of the
> page.</p> > +<p>You can create a folder or a component to help organize
> your files. To create a folder, navigate to the files tab of your project.
> Find the storage add-on you'd like to put a folder in and click the "+"
> sign next to its name. Components also act like folders, but can be used to
> control privacy settings independently of the whole project. To create a
> component, navigate to the project overview page and click “create
> component” on the right half of the page.</p> This may or may not be the
> right place for that, but we also have the Project Organizer "collections"
> now, which used to be folders. The name change isn't complete on that, but
> will be soon. I am a little concerned about how complicated it is to have
> three different types of folder-like things, but those are concerns for
> another day. — Reply to this email directly or view it on GitHub <
> https://github.com/CenterForOpenScience/osf.io/pull/2662/files#r29945131>
> .
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CenterForOpenScience/osf.io/pull/2662/files#r29956001>
> .
>